### PR TITLE
[BottomNavigation] Ensure item top and bottom content insets are identical

### PR DESCRIPTION
**Pull request by jgunaratne**
Ensure item top and bottom content insets are identical.

screen shot 2017-12-04 at 5 28 44 pm

### CODE REVIEW

**Review by brianjmoore — APPROVED** [review]

File: components/BottomNavigation/src/private/MDCBottomNavigationItemView.m [hunk 1]
```diff
@@ -139,7 +142,9 @@ - (void)centerLayoutAnimated:(bool)animated {
                     iconImageViewCenter.y - CGRectGetMidX(self.iconImageView.bounds));
     self.label.center =
         CGPointMake(CGRectGetMidX(self.bounds),
-                    CGRectGetMidY(self.bounds) + CGRectGetHeight(self.bounds) * 0.25f);
+                    CGRectGetMidY(self.bounds) + CGRectGetHeight(self.bounds) / 2 -
```
Comment (ianegordon) [inline components/BottomNavigation/src/private/MDCBottomNavigationItemView.m]: Can this line be simplified to CGRectGetHeight(self.bounds)?

File: components/BottomNavigation/src/private/MDCBottomNavigationItemView.m [hunk 2]
```diff
@@ -123,7 +124,9 @@ - (void)centerLayoutAnimated:(bool)animated {
   if (self.titleBelowIcon) {
     CGPoint iconImageViewCenter =
         CGPointMake(CGRectGetMidX(self.bounds),
-                    CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) * 0.1f);
+                    CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) / 2 +
```
Comment (ianegordon) [inline components/BottomNavigation/src/private/MDCBottomNavigationItemView.m]: Doesn't CGRectGetMidY(self.bounds) - CGRectGetHeight(self.bounds) / 2 == 0?
If so, eliminate this line?

**Review by ianegordon — APPROVED** [review]
Please consider comments before merge.

File: components/BottomNavigation/src/private/MDCBottomNavigationItemView.m (hunk 2 above)
Comment (jgunaratne) [reply components/BottomNavigation/src/private/MDCBottomNavigationItemView.m]: Done. Yay, algebra. :-)

File: components/BottomNavigation/src/private/MDCBottomNavigationItemView.m (hunk 1 above)
Comment (jgunaratne) [reply components/BottomNavigation/src/private/MDCBottomNavigationItemView.m]: Done.

**Pull request merged**